python/ray/tune/function_runner.py

```diff
@@ -507,6 +509,9 @@ def _trainable_func(self, config, reporter, checkpoint_dir):
             reporter(**{RESULT_DUPLICATE: True})
             return output
 
+        def setup_address(self):
+            return setup_address()
+
```

Comment:
this doesn't belong here (there cannot be a pytorch dependency in this class). 

You can instead implement `def execute(self, fn)`.

Reply:
Yep good point. Made the change.